marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/entity-lib.sjs

```diff
@@ -218,31 +218,12 @@ function deleteModel(entityName) {
  * @returns {[]}
  */
 function findModelReferencesInSteps(entityName, entityTypeId) {
-  let steps = [];
-
-  const mappingQuery = cts.andQuery([
-    cts.collectionQuery('http://marklogic.com/data-hub/mappings'),
-    cts.jsonPropertyValueQuery("targetEntityType", [entityName, entityTypeId])
+  const stepQuery = cts.andQuery([
+    cts.collectionQuery('http://marklogic.com/data-hub/steps'),
```

Comment:
I think this fixes another bug which is that the original query was querying on the mappings collection, but that will pick up existing legacy mappings that have not yet been deleted. And we don't care about those.